README.md

```diff
@@ -1,2 +1,15 @@
-# choreo-sample-todo-list-app
-Choreo Sample - Todo List Next.js Web App
+# Next.js Todo List Application
+
+This is a simple fullstack Next.js Todo list application that can be used to demonstrate WebApp deployments of Choreo. This application utilizes Typescript, React and Tailwind CSS
+
+## Deploy In Choreo
+
+- Fork this repository.
+- Please refer to the Choreo documentation under the [Develop a Web Application](https://wso2.com/choreo/docs/develop-components/develop-a-web-application/) section to learn how to deploy the application.
```

Comment:
Suggestion:
For instructions on how to deploy the web application, see the [Develop a Web Application](https://wso2.com/choreo/docs/develop-components/develop-a-web-application/) page in the Choreo documentation.